lib/pharos/kubeconfig_command.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'pharos/ssh/client'
+require 'pharos/yaml_file'
+require 'pharos/terraform/json_parser'
+require 'pharos/kube/config'
+
+module Pharos
+  class KubeconfigCommand < Pharos::Command
+    option ['-c', '--config'], 'PATH', 'Path to config file (default: cluster.yml)', attribute_name: :config_yaml
+
+    option '--tf-json', 'PATH', 'Path to terraform output json' do |config_path|
+      begin
+        File.realpath(config_path)
+      rescue Errno::ENOENT
+        signal_usage_error 'File does not exist: %<path>s' % { path: config_path }
+      end
+    end
+
+    option ['-n', '--name'], 'NAME', 'Overwrite cluster name', attribute_name: :new_name
+    option ['-C', '--context'], 'CONTEXT', 'Overwrite context name', attribute_name: :new_context
+
+    option ['-i', '--input'], '[FILE]', 'Use local file as input instead of remote'
```

Comment:
`--input` is a bit confusing.. probably we don't need it because merging local files can be done with `kubectl`?